astropy/io/votable/exceptions.py

```diff
@@ -1135,6 +1135,19 @@ class W55(VOTableSpecWarning):
     default_args = ("", "")
 
 
+class W56(VOTableSpecWarning):
+    """
+    The column fields as defined using ``FIELD`` elements do not match
+    those in the headers of the embedded PARQUET file.  If ``verify`` is not
+    ``'exception'``, the embedded PARQUET file will take precedence.
+    """
+
+    message_template = (
+        "The fields defined in the VOTable do not match those in the "
+        + "embedded PARQUET file"
```

Comment:
to please ruff, use implicit concatenation
```suggestion
        "embedded PARQUET file"
```

Reply:
done